docs-react-native/react-native/docs/ios/remote-notification-support.md

```diff
@@ -95,79 +76,141 @@ At this point everything should still be running normally. This is the final ste
 ```
 <!-- <Vimeo id="remote-notification-support-3" caption="Step 3 - Edit NotificationService.m" /> -->
 
-Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 
```

Comment:
I like this suggestion for people to do a build test? We used to get a lot of problems from people where they had a mix of "did not even add the extension correctly" and "added custom code that did not work"

It was useful to separate the concerns and make sure they added things correctly before they jumped in to custom code

Should probably make that suggestion *after* we suggest to add the import as the build test will then make sure they have altered their Podfile correctly to have the dependency on the notifee pod and everything is linked up correctly

Reply:
> It was useful to separate the concerns and make sure they added things correctly before they jumped in to custom code.

Hmm, that's a fair point.

However I decided to leave this "build and run app" step after all the native code changes because some people have some very slow machines and running multiple builds can be frustrating. Errors should be pretty descriptive, but it's up to you, reverting this line will be easy